cluster/libvirt-coreos/user_data_master.yml

```diff
@@ -16,7 +16,7 @@ coreos:
         ExecStart=/opt/kubernetes/bin/kube-apiserver \
         --address=0.0.0.0 \
         --port=8080 \
-        --etcd_servers=${etcd_servers} \
+        --etcd_servers=http://127.0.0.1:4001 \
```

Comment:
The Salt ones I'm comfortable with. Do you know if `libvirt-coreos` is launching the etcd pod yet? I suppose this shouldn't matter anyways, though, since it's running regardless, but in true Core style, it looks like libvirt-coreos is setting up an etcd cluster, too? https://github.com/GoogleCloudPlatform/kubernetes/blob/master/cluster/libvirt-coreos/util.sh#L194